src/App.tsx

```diff
@@ -1,14 +1,44 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { getTodos, USER_ID } from './api/todos';
+import { TodoList } from './TodoList/TodoList';
+import { Todo } from './types/Todo';
+import classNames from 'classnames';
+import { Filter } from './types/FilterButton';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState('');
+
+  const [filter, setFilter] = useState<Filter>(Filter.All);
+
+  useEffect(() => {
+    getTodos()
+      .then(setTodos)
+      .catch(() => setError('Unable to load todos'));
+    setTimeout(() => setError(''), 3000);
```

Comment:
What if data will be loading more than 3 seconds, then your message will never disappear